packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -17,14 +17,10 @@ export const BlockLoaderInput: React.VFC = () => {
   const [blockUrl, setBlockUrl] = useState("");
   const blockUrlRef = useRef<HTMLInputElement | null>(null);
 
-  const isDefinedBlock = userBlocks.some(
-    (userBlock) =>
-      createNormalizedBlockUrl(userBlock.componentId) ===
-      createNormalizedBlockUrl(blockUrl),
-  );
+  const isDefinedBlock = !!userBlocks[createNormalizedBlockUrl(blockUrl)];
```

Comment:
`userBlocks` is now a map - see changes to `packages/hash/frontend/src/blocks/userBlocks.tsx`